xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/schema.xml

```diff
@@ -242,7 +242,7 @@
    <dynamicField name="*_ar"  type="text_ar"    indexed="true"  stored="true" multiValued="true" />
    <dynamicField name="*_bg"  type="text_bg"    indexed="true"  stored="true" multiValued="true" />
    <dynamicField name="*_ca"  type="text_ca"    indexed="true"  stored="true" multiValued="true" />
-   <dynamicField name="*_cz"  type="text_cz"    indexed="true"  stored="true" multiValued="true" />
+   <dynamicField name="*_cs"  type="text_cz"    indexed="true"  stored="true" multiValued="true" />
```

Comment:
@Enygma2002 I'm not sure about this. The dynamic field is part of our customizations (so it's ok to change it) but the type definition comes from Solr and we didn't modify it so far. Changing it might make the upgrade a little more difficult (e.g. the person doing the upgrade might keep both ``text_cz`` and ``text_cs``). Moreover, as I commented on [XWIKI-11897](http://jira.xwiki.org/browse/XWIKI-11897) ``cz`` is the country code for Czech and maybe the Solr team convention is to use the country code on the field type definition (when there is only one language, I don't know). I prefer to keep only the customizations that are really needed.